NeverGreenGrove/NeverGreenGrove/src/DrawableTree.cpp

```diff
@@ -36,11 +36,15 @@ std::shared_ptr<TreeB::TreeObj> TreeB::TreeObj::s_Instance;
 
 TreeA::TreeObj::TreeObj()
 {
+	Texture temp("assets/wall.jpg");
```

Comment:
this is not the proper way of instantiating an object with a no default ctor... the correct was is to use an [initializer list](http://en.cppreference.com/w/cpp/language/initializer_list) this case would be like...
```c++
TreeA::TreeObj::TreeObj() : m_Texture("assets/wall.jpg") 
```

Reply:
This, I didn't know. In my original commit, I wrote a comment asking if there's a better way to do this...now i know how!